locales/en/index.yml

```
@@ -3336,7 +3336,8 @@ idpay:
             showMore: Vedi tutte
             header: Impostazioni
             associatedPaymentMethods: Metodi di pagamento associati
-            methodsi18n: metodi 
+            methodsPlural: metodi 
+            methodsSingular: metodo 
```

You could use the pluralize functionality of i18n-js

```suggestion
            methods:
              one: "1 metodo"
              other: "{{count}} metodi"
```